feature/session/src/main/java/io/github/droidkaigi/confsched2020/session/ui/SearchSessionsFragment.kt

```diff
@@ -148,10 +153,27 @@ class SearchSessionsFragment : Fragment(R.layout.fragment_search_sessions), Inje
         }
     }
 
+    override fun onResume() {
+        super.onResume()
+        val intent = activity!!.intent
+        if (intent.action == Intent.ACTION_SEARCH) {
+            val query: String = intent.getStringExtra(SearchManager.QUERY)!!
```

Comment:
```suggestion
            val query: String = requrireNotNull(intent.getStringExtra(SearchManager.QUERY))
```

Reply:
I took it!
But i didn't notice typo💦
I fixed it 4998c258a09acc3275a89ea01fb4f4f2d5ae33a9

Reply:
🙇